ui/src/js/project/planting/ProjectPlantingWithoutSlider.js

```diff
@@ -6,7 +6,6 @@ import {Link} from 'react-router';
 import axios from 'axios';
 
 import IconButton from '../../common/components/IconButton';
-import ImageButton from '../../common/components/ImageButton';
```

Comment:
Is the ImageButton component still used? If not please remove it from the code base too. Thanks.

Reply:
No, I did not recognize that occurence, thanks. I performed a fulltext search over the ui folder, I hope all other compoments not used anymore are removed.